### PR TITLE
minecraft updates

### PR DESCRIPTION
**Pull request by dalecassidy**
These are a couple of changes needed to get minecraft working due to python api changes

### CODE REVIEW

File: opencog/cython/opencog/spatial.pyx [hunk 1]
```diff
@@ -105,10 +105,9 @@ cdef class EntityRecorder:
     def __cinit__(self):
         pass
 
-    def __init__(self, long addr, AtomSpace atomspace):
+    def __init__(self, long addr):
         self.c_entity_recorder = <cEntityRecorder*> PyLong_AsVoidPtr(addr)
-        self.atomspace = atomspace
-
+
```
Comment (linas) [inline opencog/cython/opencog/spatial.pyx]: are you sure?  This now differs from how `__init__` works for OctomapOCTree up above. Also, it looks like now, self.atomspace is never initialized; yet it is used by more-or-less every method in this class ...

Atoms, once inserted into some atomspace, always know what atomspace they are in.  Only fresh atoms, that have never been inserted anywhere, don't know where they belong., so they're left floating about.   Maybe the atomspace argument isn't needed for most of these functions? Maybe passing an invalid atomspace is OK ???

Comment (linas) [inline opencog/cython/opencog/spatial.pyx]: p.s. I'm not an expert at his stuff, someone  else should review and commit as needed. This just caught my eye.

Comment (dalecassidy) [inline opencog/cython/opencog/spatial.pyx]: You may be right. I'll take a look at it again.

**Comment by linas** [comment]
rejecting, this appears to be super-ceeded by pull req #2090

**Comment by linas** [comment]
oh, wait, it looks like this branch was also a part of #2090 and got merged when that one got merged.

**Pull request merged**